build.sbt

```diff
@@ -5,3 +5,7 @@ version := "0.1"
 scalaVersion := "2.13.1"
 
 libraryDependencies += "org.scalatest" % "scalatest_2.13" % "3.0.8" % "test"
```

Comment:
put this dependency into the Seq one line below, no point in doing it two ways like this